app/controllers/workflows_controller.rb

```diff
@@ -91,6 +92,14 @@ def history
 
   private
 
+  # Fetches the workflow from the workflow service and checks to see if it's active
+  def workflow_active?(wf_name)
+    xml = Dor::Config.workflow.client.get_workflow_xml('dor', @object.pid, wf_name)
+    ng_xml = Nokogiri::XML(xml)
+    result = ng_xml.at_xpath("/workflow/process[@version=#{@object.current_version}]")
+    result ? true : false
+  end
+
```

Comment:
could you do this to avoid having to directly parse workflow XML outside of the workflow service? ```Dor::Config.workflow.client.get_active_workflows('dor', @object.pid).include? wf_name```

Reply:
I don't think so, because that depends on `archived` behavior, which is something we removed from the new workflow service.

Reply:
ok

Reply:
how about moving this into a new workflow service method ?  don't feel strongly enough to block progress on this PR, but seems cleaner and removes specific XML knowledge from a consumer

Reply:
Ideally the workflow client could answer this question, yeah. I too don't care too strongly.

Reply:
I think that sounds good.  Earlier I was hoping we could move all the workflow models there, but I'm waiting on https://github.com/sul-dlss/workflow-server-rails/issues/102, first. However, I think it wouldn't be difficult to model the response.